src/theme.ts

```diff
@@ -21,27 +21,27 @@ const colors = {
 	},
 
 	danger: {
-		main: '#FF4444',
-		semilight: '#FFA1A1',
-		light: '#FFECEC',
-		dark: '#661B1B',
+		main: '#ff4444',
+		light: '#FFA1A1',
+		extraLight: '#FFECEC',
+		dark: '#661B1b',
 	},
 	warning: {
 		main: '#FCA321',
-		semilight: '#FDD190',
-		light: '#FEF5E9',
-		dark: '#64410D',
+		light: '#FDD190',
+		extraLight: '#FEF5E9',
+		dark: '#64410d',
 	},
 	success: {
 		main: '#4DB313',
-		semilight: '#A6D989',
-		light: '#EDF7E7',
-		dark: '#1E4707',
+		light: '#A6D989',
+		extraLight: '#EDF7E7',
+		dark: '#1e4707',
 	},
 	info: {
 		main: '#2297DE',
-		semilight: '#90CBEE',
-		light: '#E9F4FB',
+		light: '#90CBEE',
+		extraLight: '#E9F4FB',
```

Comment:
Oh wait. These are breaking changes here! These are exported and we use them in the UI.
Please reconsider not changing the names if it makes sense for now.
If we do want to rename these, then please also update the typings.

Reply:
Ah I see, though we only use them in two places in 2FA.

Reply:
Unfortunately the colours before had big variance for something called semilight, so it would be best to keep it somehow consistent.

Do we really want to mark it as major when the UI change is so minor?

Reply:
@dimitrisnl Can you change this to a major version bump and add a migration doc please? I'm not opposed to doing major bumps on small things like this, we just need to be clear about what pieces of the API changed

Reply:
Btw I'm not using it the new properties in this PR so I could discard these changes.

Reply:
@dimitrisnl If you'd like to discard these changes, for now, that's fine as well. You could make an issue to review the color swatches for v5

Reply:
Sure this would be for the best @LucianBuzzo . Cherry picking the changes atm